src/Select.tsx

```diff
@@ -14,10 +23,16 @@ export type SelectProps = Partial<Omit<_SelectProps, 'children' | 'renderer'>> &
   }>;
 
 function Select(props: SelectProps, ref: ForwardedRef<SelectElement>): ReactElement | null {
+  const innerRef = useRef<SelectElement>(null);
   const [portals, renderer] = useSimpleOrChildrenRenderer(props.renderer, props.children);
+  const finalRef = useMergedRefs(innerRef, ref);
+
+  useEffect(() => {
+    innerRef.current?.requestContentUpdate();
```

Comment:
I think it should only request a content update when a renderer is provided. When items API is used, everything already works correctly.

Reply:
Oh, right, I forgot about that point